index.html

```diff
@@ -36,6 +36,192 @@ <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
+                        </ul>
+        
+                        <button class="card-btn">See Project</button>
+        
+                    </div>
+        
+        
+                    </div>
+
+
+                    <div class="card">
+                        <div class="card-items">
+                            <img src="img/work-1.png" alt="" class="card-img">
+                            <h2 class="card-name">Multi-Post Stories</h2>
+                            <ul class="card-props">
+                                <li  class="card-prop"><a href="#">Canopy</a></li>
+                                <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                                <li class="card-prop"><a href="#">Back End Dev</a></li>
+                                <li class="card-prop"><a><img src="img/Counter.png"></a></li>
+                                <li  class="card-prop"><a href="#">Canopy</a></li>
+            
+                            </ul>
+            
+                            <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            
+                            <ul class="card-languages">
+                                <li class="card-language">HTML</li>
+                                <li class="card-language">CSS</li>
+                                <li class="card-language">Javascript</li>
+                            </ul>
+            
+                            <button class="card-btn">See Project</button>
+            
+                        </div>
+            
+            
+                        </div>
+
+            
+        </div>
+    </section>
+
+    <section class="about-section">
+        <div class="about-me">
+
+        <div class="about-items">
+            <h2 class="about-heading">About Myself</h2>
+            <p class="about-text">
+                Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <div class="connect">
+                <p class="connect-text">lets connect</p>
+                <ul class="connect-icons">
+                    <li><a href="https://twitter.com/JosiahMutinhiri"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/josiah-mutinhiri-536387153/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-sharp fa-solid fa-m"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-brands fa-angellist"></i></a></li>
+        
+                </ul>
+            </div>
+
```

Comment:
Your design
![image](https://github.com/josman227/Mobile-Portfolio/assets/78251927/9e1cc7be-9e83-4612-94ac-636d26c4408d)

Figma
![image](https://github.com/josman227/Mobile-Portfolio/assets/78251927/5096c539-9648-46ee-be6c-0aadf61ce905)


- Please kindly remove the grey border in the `About` as it is not present in the original design so as to stick to the given design as much as possible. 🤗

- To include the `border-radius` could you notice the background color of the parent element 👀🛠 🤗